src/Components/ProvisioningWizard/helpers.js

```diff
@@ -1,63 +1,12 @@
 import PropTypes from 'prop-types';
-import { useQuery, useQueries } from 'react-query';
 
-import { sourcesQueryKey, sourceUploadInfoKey } from '../../API/queryKeys';
-import { fetchSourcesList, fetchSourceUploadInfo } from '../../API';
-import { AWS_PROVIDER, AZURE_PROVIDER } from '../../constants';
+import { AWS_PROVIDER, AZURE_PROVIDER, GCP_PROVIDER } from '../../constants';
 
 export const imageProps = PropTypes.shape({
   name: PropTypes.string,
   id: PropTypes.string,
-  provider: PropTypes.string,
+  provider: PropTypes.oneOf([AWS_PROVIDER, AZURE_PROVIDER, GCP_PROVIDER]),
   architecture: PropTypes.string,
   sourceIDs: PropTypes.arrayOf(PropTypes.string),
   accountIDs: PropTypes.arrayOf(PropTypes.string),
 }).isRequired;
-
-export const sourcesData = (image, { refetch = false } = {}) => {
```

Comment:
moved to Hooks and renamed to `useXXX` so it can be properly checked by linters.